logger/zap.go

```diff
@@ -0,0 +1,62 @@
+package logger
+
+import (
+	"log"
+	"os"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var zapLogger *zap.Logger
+
+func init() {
+	logLevelEnv := os.Getenv("LOG_LEVEL")
+	logLevel := zapcore.DebugLevel
```

Comment:
Since the following switch statement also handles the default case, you need not assign a default value to `logLevel` here.
This line should be replaced with `var logLevel zapcore.Level`.

Reply:
Fix default value
https://github.com/datachainlab/yui-relayer/pull/4/commits/30d32e2c9d818ea9c7c8010c6aa047e7ccbe3cb5